mscore/scoreview.cpp

```diff
@@ -34,6 +34,7 @@
 #include "textcursor.h"
 #include "textpalette.h"
 #include "texttools.h"
+#include <qtoolbar.h>
```

Comment:
Yes. Qt headers are included in all.h which is used for the precompiled header optimisation (on linux and macosx).